distribution/docs/src/main/resources/content/_coreConcepts/high-availability-intro.adoc

```diff
@@ -0,0 +1,47 @@
+:type: coreConcept
+:status: published
+:title: Introduction to High Availability
+:order: 10
+
+${branding} can be made to be highly available. In this context, High Availability is defined as the ability for the ${branding} to be continuously operational with very little down time.
+
+In a Highly Available Cluster, the ${branding} has failover capabilities when a system fails.
+This setup consists of a Solr Cloud instance, 2 ${branding} nodes connected to that Solr Cloud, and a load balancer that sits in front of those 2 nodes.
+One of the ${branding} nodes will be arbitrarily chosen to be the active node, and the other will be the standby node.
```

Comment:
❗️ qualify standby as "hot standby" since it is up and ready to process requests right away.